node/router/router.go

```diff
@@ -180,18 +183,27 @@ func createRouter(shardIDs []types.ShardID, batcherEndpoints map[types.ShardID]s
 		shardIDs:         shardIDs,
 		routerNodeConfig: rconfig,
 		verifier:         verifier,
+		stopChannel:      make(chan bool, 1),
 	}
 
 	for _, shardId := range shardIDs {
 		r.shardRouters[shardId] = NewShardRouter(logger, batcherEndpoints[shardId], batcherRootCAs[shardId], rconfig.TLSCertificateFile, rconfig.TLSPrivateKeyFile, rconfig.NumOfConnectionsForBatcher, rconfig.NumOfgRPCStreamsPerConnection, verifier)
 	}
 
 	go func() {
+		r.logger.Debugf("statistics reporter in router is starting")
```

Comment:
change comment, this is not a statistics reporter 
also I think it is safe to make it info instead of debug